Breezy-Traveler/Once.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Once.swift
+//  Breezy-Traveler
+//
+//  Created by Erick Sanchez on 12/13/18.
+//  Copyright © 2018 Phyllis Wong. All rights reserved.
+//
+
```

Comment:
This file helps ensure a block of code is only executed once unless the `reset` function is used to reset the variable.

Reply:
We should probably go through the whole Xcode app and change where it says Copyright © 2018 Phyllis Wong to Copyright © 2018 Breezy Traveler